modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -3,29 +3,338 @@
+	const [activeBreadcrumb, setActiveBreadcrumb] = useState(0);
+	const [activeTab, setActiveTab] = useState(0);
+
 	return (
-		<ClayLayout.Sheet className="mt-3" size="lg">
-			<ClayLayout.SheetHeader>
-				<h2 className="sheet-title">
-					{Liferay.Language.get('actions')}
-				</h2>
-			</ClayLayout.SheetHeader>
-
-			<ClayLayout.SheetFooter>
-				<ClayButton.Group spaced>
-					<ClayButton>{Liferay.Language.get('save')}</ClayButton>
-
-					<ClayButton displayType="secondary">
-						{Liferay.Language.get('cancel')}
-					</ClayButton>
-				</ClayButton.Group>
-			</ClayLayout.SheetFooter>
-		</ClayLayout.Sheet>
+		<ClayLayout.ContainerFluid>
+			<ClayBreadcrumb
+				items={[
+					{
+						active: activeBreadcrumb === 0,
+						href: '#1',
+						label: Liferay.Language.get('actions'),
+					},
+					{
+						active: activeBreadcrumb === 1,
+						href: '#2',
+						label: Liferay.Language.get('new-item-action'),
+					},
+				]}
+			/>
+
+			<ClayLayout.ContainerFluid className="bg-white mb-4 p-0 rounded-sm">
+				{activeBreadcrumb === 0 && (
+					<>
+						<h2 className="mb-0 p-4">
+							{Liferay.Language.get('actions')}
+						</h2>
+
+						<ClayTabs
+							active={activeTab}
+							onActiveChange={setActiveTab}
+						>
+							<ClayTabs.Item>
+								{Liferay.Language.get('item-actions')}
+							</ClayTabs.Item>
+
+							<ClayTabs.Item>
+								{Liferay.Language.get('creation-actions')}
+							</ClayTabs.Item>
+
+							<ClayTabs.Item>
+								{Liferay.Language.get('bulk-actions')}
+							</ClayTabs.Item>
+						</ClayTabs>
+
+						<ClayTabs.Content active={activeTab} fade>
+							<ClayTabs.TabPane aria-labelledby="tab-1">
+								<OrderableTable
+									className="mt-0 p-1"
+									fields={[
+										{
+											label: Liferay.Language.get('icon'),
+											name: 'icon',
+										},
+										{
+											label: Liferay.Language.get(
+												'label'
+											),
+											name: 'label',
+										},
+										{
+											label: Liferay.Language.get('type'),
+											name: 'type',
+										},
+									]}
+									items={[]}
+									noItemsButtonLabel={Liferay.Language.get(
+										'create-item-action'
+									)}
+									noItemsDescription={Liferay.Language.get(
+										'start-creating-one-action-to-interact-with-your-data'
+									)}
+									noItemsTitle={Liferay.Language.get(
+										'no-actions-were-created'
+									)}
+									onCancelButtonClick={noop}
+									onCreationButtonClick={() =>
+										setActiveBreadcrumb(1)
+									}
+									onOrderChange={noop}
+									onSaveButtonClick={noop}
+								/>
+							</ClayTabs.TabPane>
+
+							<ClayTabs.TabPane aria-labelledby="tab-2">
+								2. Proin efficitur imperdiet dolor, a iaculis
+								orci lacinia eu.
+							</ClayTabs.TabPane>
+
+							<ClayTabs.TabPane aria-labelledby="tab-3">
+								3. Proin efficitur imperdiet dolor, a iaculis
+								orci lacinia eu.
+							</ClayTabs.TabPane>
```

Comment:
I would not even put a placeholder for bulk actions here. It is planned for Q4 I believe, so we will need to delete this later on or put behind new FF :grimacing: I also don't see much sense in bulk actions as a separate configuration category, they are basically item actions. We need to add support for them in FDS, sure, but I don't see any configuration additional to item actions that we would need to add here.